dll/win32/netid/netid.c

```diff
@@ -180,13 +180,15 @@ IsUserAdmin(VOID)
         goto done;
 
     dwSize = 0;
-    GetTokenInformation(hToken,
-                        TokenGroups,
-                        NULL,
-                        0,
-                        &dwSize);
-    if (dwSize == 0)
+    if (GetTokenInformation(hToken,
```

Comment:
`GetTokenInformation` returns a BOOL value. With your change the code literally hits the check when this function successfully retrieves details about the token....

Reply:
This is how it supposed to be used: first you call `GetTokenInformation` with NULL buffer and expect it to return `ERROR_INSUFFICIENT_BUFFER`. Then you make a proper call